Dockerfile

```
@@ -0,0 +1,39 @@
+
+FROM openjdk:17-jdk-slim
+
+# --- 런타임 필수 패키지 설치 --------------------------
+RUN apt-get update && \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)